viewing_party/party.py

```diff
@@ -1,23 +1,183 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+
+    new_movie = {}

```

Comment:
I suggest moving the `new_movie` declaration inside the if-statement so that we don't allocate space for `new_movie` unless we know it will be used.

Reply:
That makes sense from both a memory/space perspective and a cleaner code/logic perspective. Thank you!